suites/krbd/rbd/tasks/rbd_fio.yaml

```diff
@@ -3,7 +3,7 @@ tasks:
 - ceph: null
 - rbd_fio:
     client.0:
-       fio-version: 2.2.9
+       fio-version: 2.2.11
```

Comment:
I'd actually suggest yanking these overrides from all the test case YAML files and instead update the default in rbd_fio.py.